crates/bevy_macro_utils/src/lib.rs

```diff
@@ -111,7 +112,30 @@ impl BevyManifest {
 /// - `input`: The [`syn::DeriveInput`] for struct that is deriving the label trait
 /// - `trait_path`: The path [`syn::Path`] to the label trait
 pub fn derive_label(input: syn::DeriveInput, trait_path: &syn::Path) -> TokenStream {
-    let ident = input.ident;
+    // return true if the variant specified is an `ignore_fields` attribute
+    fn is_ignore(attr: &syn::Attribute, attr_name: &str) -> bool {
+        if attr.path.get_ident().as_ref().unwrap() != &attr_name {
+            return false;
+        }
+
+        syn::custom_keyword!(ignore_fields);
+        attr.parse_args_with(|input: syn::parse::ParseStream| {
+            let ignore = input.parse::<Option<ignore_fields>>()?.is_some();
+            Ok(ignore)
+        })
+        .unwrap()
+    }
+
+    let ident = input.ident.clone();
+
+    use convert_case::{Case, Casing};
+    let trait_snake_case = trait_path
+        .segments
+        .last()
+        .unwrap()
+        .ident
+        .to_string()
+        .to_case(Case::Snake);
```

Comment:
Rather than pull in a crate to convert PascalCase to snake_case, i think i would prefer to just manually pass in a string constant to `derive_label`.